src/sql/src/session/vars.rs

```diff
@@ -621,6 +621,15 @@ pub const ENABLE_RBAC_CHECKS: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+pub const FORCE_INTROSPECTION_CLUSTER: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("force_introspection_cluster"),
```

Comment:
Would love to bikeshed this name a bit; `force_introspection_cluster` was just a strawman that I don't love. Other options include `auto_use_introspection_cluster` or something like that. Maybe start a discussion in the devex channel?

```suggestion
    name: UncasedStr::new("auto_use_introspection_cluster"),
```